flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -75,6 +76,8 @@ function verify_result {
 }
 
 function shutdown_elasticsearch_cluster {
+   local index=$1
```

Comment:
adjust existing usages in `test_streaming_elasticsearch.sh`

Reply:
+1